### DISCUSSION TITLE
Does material theme provids the visitor counters?

### DISCUSSION THREAD

**Discussion by hellojinwoo** [squidfunk/mkdocs-material#5431] [Q&A]
Does material theme provide features "visitors counter", which shows the total number of visitors or the number of visitors during today?

**Comment by squidfunk** [comment]
Nope, but you can embed any JavaScript-based third-party solution using [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript).

**Reply by hellojinwoo** [reply]
- Sorry for bothering you, but can you kindly point out the tutorial for learning how to add the visitor counter?
- If you recommend one source that is easy to integrate to mkdocs material theme, it would be super nice for a code newbie like me.

![image](https://user-images.githubusercontent.com/34431729/234991596-7f3387ef-decd-4527-9b50-c69c8f65f8bd.png)
